test/e2e/clusterctl_upgrade_test.go

```diff
@@ -325,3 +326,31 @@ var _ = Describe("When testing clusterctl upgrades using ClusterClass (v1.7=>cur
 		}
 	})
 })
+
+var _ = Describe("When testing clusterctl upgrades using ClusterClass (v1.7=>current) on latest management cluster [ClusterClass]", func() {
```

Comment:
```suggestion
var _ = Describe("When testing clusterctl upgrades using ClusterClass (v1.7=>current) on K8S latest ci mgmt cluster [ClusterClass]", func() {
```
Just for consistency with the other test using that version